t

```diff
@@ -0,0 +1,31 @@
+[1mdiff --git a/src/vivarium_census_prl_synth_pop/constants/data_values.py b/src/vivarium_census_prl_synth_pop/constants/data_values.py[m
+[1mindex 82e883f..9211c28 100644[m
+[1m--- a/src/vivarium_census_prl_synth_pop/constants/data_values.py[m
+[1m+++ b/src/vivarium_census_prl_synth_pop/constants/data_values.py[m
+[36m@@ -96,8 +96,8 @@[m [mYEARLY_JOB_CHANGE_RATE = 0.5  # 50 changes per 100 py[m
+ [m
+ BUSINESS_MOVE_RATE_YEARLY = 0.1  # 10 changes per 100 py[m
+ [m
+[31m-PERSONAL_INCOME_PROPENSITY_DISTRIBUTION = stats.norm(loc=0.0, scale=0.812309 ** 0.5)[m
+[31m-EMPLOYER_INCOME_PROPENSITY_DISTRIBUTION = stats.norm(loc=0.0, scale=0.187691 ** 0.5)[m
```

Comment:
delete this file

Reply:
We don't need this anymore?

Reply:
Isn't it a junk file? The filename is just "t" and it appears to be a `git diff`